pkgs/development/python-modules/scalene/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, setuptools-scm
+, cloudpickle
+, cython
+, jinja2
+, psutil
+, pynvml
+, pythonOlder
+, rich
+}:
+
+buildPythonPackage rec {
+  pname = "scalene";
+  version = "1.5.35";
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-3s0KN06RivCN4G5/LZf+r7GtAlKwqRuZwUtrY2T2Bew=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-warn "pynvml>=11.0.0,<11.5" "pynvml~=11.5.0"
+    substituteInPlace requirements.txt \
+      --replace-warn "pynvml~=11.0.0" "pynvml~=11.5.0"
+  '';
+
+  nativeBuildInputs = [
+    cython
+    setuptools
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    cloudpickle
+    jinja2
+    psutil
+    pynvml
+    rich
+  ];
+
+  pythonImportsCheck = [ "scalene" ];
+
+  meta = with lib; {
+    description = "Scalene: A high-resolution, low-overhead CPU, GPU, and memory profiler for Python with AI-powered optimization suggestions";
+    homepage = "https://pypi.org/project/scalene/1.5.35";
```

Comment:
```suggestion
    homepage = "https://github.com/plasma-umass/scalene";
```

Reply:
I'm getting conflicting advice. One reviewer wants a link to the specific release, ans you want it to the top of the repo. I'm finde with either so I'll let fine people negotiate.

Reply:
Nevermind, I'll post the specific link to the changelist and the general link here